docs/source/api/index.rst

```diff
@@ -0,0 +1,113 @@
+API Documentation
+=================
+
+.. toctree::
+   :titlesonly:
+   :caption: Quick Links:
+   
+   arduino
+   motor-board
+   power-board
+   servo-board
+
+   game-state
+
+Programming your robot is done in `Python <https://www.python.org/>`__,
+specifically version 3.7.4. You can learn more about Python from their
+`docs <https://docs.python.org/3/>`__, and our whirlwind tour.
+
+Setup
+-----
+
+The following two lines are required to complete initialisation of the
+kit:
+
+.. code:: python
+
+   from sbot import Robot
+
+   r = Robot()
+
+Once this has been done, this ``Robot`` object can be used to control
+the robot’s functions.
+
+The remainder of the tutorials pages will assume your ``Robot`` object
+is defined as ``r``.
+
+Running your code
+-----------------
+
+Your code needs to be put on a USB drive in a file called ``main.py``.
+On insertion into the robot, this file will be executed. The file is
+directly executed off your USB drive, with your drive as the working
+directory.
+
+To stop your code running, you can just remove the USB drive. This will
+also stop the motors and any other peripherals connected to the kit.
+
+You can then reinsert the USB drive into the robot and it will run your
+``main.py`` again (from the start). This allows you to make changes and
+test them quickly.
+
+.. Hint:: If this file is missing or incorrectly named, your
+  robot won’t do anything. No log file will be created.
+
+Start Button
+------------
+
+After the robot has finished starting up, it will wait for the *Start
+Button* on the power board to be pressed before continuing with your
+code, so that you can control when it starts moving. There is a green
+LED next to the start button which flashes when the robot is finished
+setting up and the start button can be pressed.
+
+Running Code before pressing the start button
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to do things before the start button press, such as setting
+up servos or motors, you can pass ``wait_start``. You will
```

Comment:
Perhaps "pass wait_start to the Robot constructor"? This sentence doesn't make it entirely clear where / how we're passing `wait_start`.

Reply:
There's example code just below this sentence.

However I agree that this is not great. Again copied from the current docs.

Reply:
```suggestion
up servos or motors, you can pass ``wait_start`` to the ``Robot`` constructor. You will
```